src/libsyntax/parse/parser.rs

```diff
@@ -522,7 +520,7 @@ impl<'a> Parser<'a> {
 
     /// If the next token is the given keyword, eats it and returns `true`.
     /// Otherwise, returns `false`. An expectation is also added for diagnostics purposes.
-    pub fn eat_keyword(&mut self, kw: Symbol) -> bool {
+    fn eat_keyword(&mut self, kw: Symbol) -> bool {
```

Comment:
@Centril Please reconsider before making these kinds of 'refactorings.' Tools like rustfmt which rely on the libsyntax suffer severely from reduced visibility. Now I have to iterate through the changes, re-publish hidden fields and methods, make a PR and wait until it gets merged.

I understand that the libsyntax is primarily for the rustc itself, and its interface is unstable. But please be conservative about the visibility of its interface.

Reply:
@topecongiro If you rely on something then there should be a comment to that effect in the rustc sources. Reducing visibility is an important tool for cleaning up the rustc internals and should be used.